doc/14-features.md

```diff
@@ -439,6 +439,121 @@ The recommended way of running Elasticsearch in this scenario is a dedicated ser
 where you either have the Elasticsearch HTTP API, or a TLS secured HTTP proxy,
 or Logstash for additional filtering.
 
+
+#### Elasticsearch Datastream Writer <a id="elasticsearch-datastream-writer"></a>
+
+> **Note**
+>
+> This is a newer alternative to the Elasticsearch Writer above. The Elasticsearch Datastream Writer uses
+> Elasticsearch's datastream feature and follows the Elastic Common Schema (ECS), providing better performance
+> and data organization. Use this writer for new installations. The original Elasticsearch Writer is still
+> available for backward compatibility.
+
+This feature sends check results with performance data to an [Elasticsearch](https://www.elastic.co/products/elasticsearch) instance or cluster.
+
+> **Note**
+>
+> This feature requires Elasticsearch to support timeseries datastreams and have the ecs component template installed.
+> This feature was tested successfully with Elasticsearch 8.12 and 9.0.8.
```

Comment:
Do we need to explicitly mention that the  `ElasticsearchDatastreamWriter` will not work with OpenSearch? Since the current `ElasticsearchWriter` works with OpenSearch, users might expect the same here.